todo_app/test_system.py

```diff
@@ -0,0 +1,70 @@
+import os
+import pytest
+import todo_app.app as app
+from todo_app.trello import create_board, delete_board
+from threading import Thread
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import dotenv
+import requests
+from time import sleep
+ 
+@pytest.fixture(scope='module')
+def test_app():
+
+    file_path = dotenv.find_dotenv('.env') 
+    dotenv.load_dotenv(file_path, override=True)      
+
+    # Create the new board & set it to env variable
+    board_id = create_board() 
+    os.environ['TRELLO_BOARD'] = board_id
+
+    # Get the new board list ids and update the environment variables
```

Comment:
This section for overriding the list environment variables might be nicer extracted into its own function, just for simpler reading